cli/exec.go

```diff
@@ -73,6 +74,10 @@ func ConfigureExecCommand(app *kingpin.Application) {
 		Short('s').
 		BoolVar(&input.StartServer)
 
+	cmd.Flag("ecs", "Run the ECS server in the background for credentials").
```

Comment:
my personal preference is the fully qualified flag be self-documenting, perhaps `--server-ecs` or `--ecs-server` since this command also starts a server similar to the ec2 cred provider started with `--server`

Reply:
Good point. I changed it to `--ecs-server`.

I'm not sure what need the `--server` flag and EC2 Metadata Server support would still fulfill alongside the ECS Server solution, but I suppose it would be unwelcome to remove it without a deprecation window. That said, we may not desire to make the new option have a name which is consistent with the existing option (if it results in an odd name), since it may be desirable to remove the existing option in the future.